src/useMedia.ts

```diff
@@ -0,0 +1,53 @@
+import {useState, useEffect, useLayoutEffect} from 'react';
```

Comment:
You use camel case naming this file `useMedia.ts`, but in other files you use kebab case, like `camel-to-hypen.ts`.

Reply:
Ah yea thats just a convention I follow in the projects I work on - utilities are kebab cased... but I don't need to put those conventions onto this codebase. I will rename these utilities to camelCase.